static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -271,6 +271,27 @@ $block-padding-right: $gs-gutter;
 
     .block-elements {
 
+        /* Bylines
+           ========================================================= */
+        .liveblog-block-byline {
+            border-bottom: 1px dotted #dcdcdc;
+            padding-bottom: 8px;
```

Comment:
Actually generally get @sndrs to check this over. He's better at this stuff than I am :-) But things like colours here should be using variables, and I think we have mixins for fonts.